modules/page-memory/src/main/java/org/apache/ignite/internal/pagememory/persistence/PartitionMeta.java

```diff
@@ -46,212 +45,33 @@ public class PartitionMeta {
         }
     }
 
-    private volatile long lastAppliedIndex;
-
-    private volatile long lastAppliedTerm;
-
-    private volatile long leaseStartTime;
-
-    private volatile long lastReplicationProtocolGroupConfigFirstPageId;
-
-    private volatile long freeListRootPageId;
-
-    private volatile long versionChainTreeRootPageId;
-
-    private volatile long indexTreeMetaPageId;
-
-    private volatile long gcQueueMetaPageId;
+    protected volatile PartitionMetaSnapshot metaSnapshot;
```

Comment:
It seems that this field can be `private`

Reply:
done